### ISSUE TITLE
Idea: add a `limbo` subcommand that generates a reasonably formatted GitHub issue

### ISSUE THREAD

**Issue by woodruffw** [C2SP/x509-limbo#12]
Random thought: this will be nice to have when reporting bugs upstream:

```bash
limbo explain testcast-id
```

...should produce a Markdown formatted explanation of what `testcase-id` does, including all of the relevant states/chains required for the upstream to reproduce it.

Or maybe this would be overkill and/or should be done somewhere else. Just saving this as an issue so I don't forget it.

**Comment by tnytown** [comment]
This sounds reasonable! We might also want to add links to specific subsections of the spec that's being violated to the test descriptions.

**Comment by woodruffw** [comment]
Yep, let's do that for new testcases going forwards -- the docstring for each testcase seems like a reasonable place to put that.
